src/database/DataTypes/TransactionBatch.js

```diff
@@ -118,6 +114,7 @@ export class TransactionBatch extends Realm.Object {
     }
     // Must be a supplier invoice.
     if (!this.costPrice) return 0;
+    if (this.transaction?.type === 'supplier_credit') return -this.costPrice * this.numberOfPacks;
```

Comment:
totalPrice in mobile are a `getter` on the object - for a supplier credit this will always be negative